### ISSUE TITLE
look -> 'Lookined' in home page

### ISSUE THREAD

**Issue by ildella** [spencermountain/compromise#543]
```  
const doc = nlp('I’m lookin for Amanda Hugginkiss')
const past = doc.sentences().toPastTense().out('text')
console.log(past)
```
output -> "i lookined for Amanda Hugginkiss"
I tried this as is one of the examples [on the website](http://compromise.cool/)

**Comment by spencermountain** [comment]
oops!
thanks Daniele, I should have caught that!

**Comment by spencermountain** [comment]
got it fixed in `11.12.4`
thanks
